app/helpers/users_helper.rb

```diff
@@ -21,21 +21,17 @@ def print_pseuds(user)
   end
 
   # Determine which icon to show on user pages
-  def standard_icon(user = nil, pseud = nil)
-    if pseud && pseud.icon
-      pseud.icon.url(:standard).gsub(/^http:/, "https:")
-    elsif user && user.default_pseud && user.default_pseud.icon
-      user.default_pseud.icon.url(:standard).gsub(/^http:/, "https:")
-    else
-      '/images/skins/iconsets/default/icon_user.png'
-    end
+  def standard_icon(pseud = nil)
+    return pseud.icon.variant(:standard).processed.url if pseud&.icon&.attached?
```

Comment:
Oops, missed this comment from the review.

The second safe navigation looks unneeded.
```suggestion
    return pseud.icon.variant(:standard).processed.url if pseud&.icon.attached?
```

Also, this method is arranged differently than the one in collections_helper. It will probably make our lives easier in the future if they're both arranged in the same order.